src/js/login/reducers/login.js

```diff
@@ -35,13 +37,13 @@ function loginStuff(state = initialState, action) {
       return _.set('currentlyLoggedIn', action.value, state);
 
     case UPDATE_LOGIN_URL:
-      return _.set('loginUrl', action.value, state);
+      return _.set('loginUrl', appendQuery(action.value, { clientId: gaClientId() }), state);
 
     case UPDATE_VERIFY_URL:
-      return _.set('verifyUrl', action.value, state);
+      return _.set('verifyUrl', appendQuery(action.value, { clientId: gaClientId() }), state);
```

Comment:
Reducers are supposed to be pure functions in Redux, it would be better if you could call `gaClientId()` in the action creator functions for these and pass the id in on the action object.